components/script/dom/htmliframeelement.rs

```diff
@@ -255,9 +255,6 @@ impl HTMLIFrameElement {
 
         self.pipeline_id.set(Some(new_pipeline_id));
 
-        let mut blocker = self.load_blocker.borrow_mut();
```

Comment:
Should we be terminating the load blocker when the frame content changes due to history traversal?